bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -402,6 +407,11 @@ dn: cn=config
 changetype: modify
 add: olcDisallows
 olcDisallows: bind_anon
+
+dn: cn=config
+changetype: modify
+add: olcRequires
+olcRequires: authc
```

Comment:
Why is this change necessary?

Reply:
Although the wording of the README specifies that `LDAP_ALLOW_ANON_BINDING` targets the binding functionality, it seemed to me that this would be mainly used to prevent anonymous access to the directory. While I was configuring my instance I stumbled across [OpenLDAP Configuration Reference, 14.3.1](https://www.openldap.org/doc/admin26/guide.html#%22simple%22%20method) which suggests that simply disallowing bind_anon does not actually prevent anonymous access.
I am unaware under what circumstances this is the case or whether this applies to some data accessible without binding to an identity, but it felt to have the possibility of setting false security expectations to me.
However I could see reason for not including this into `LDAP_ALLOW_ANON_BINDING` but rather have it be a dedicated setting with a note for reference.